erpnext/utilities/order_for.py

```diff
@@ -0,0 +1,81 @@
+import frappe
+from erpnext.shopping_cart.doctype.shopping_cart_settings.shopping_cart_settings import get_shopping_cart_settings
+
+@frappe.whitelist(allow_guest=False)
```

Comment:
```suggestion
@frappe.whitelist()
```

Reply:
Yeah, I don't think I want to allow a guest user to touch anything to do with customer setting until we figure out what to do with guest users and shopping cart actions.

Reply:
Meaning I want to make sure next person who sees this code are super aware that we want to disallow guest access so that's why I left allow=guest=False to make it super obvious